packages/@okta/vuepress-site/docs/reference/api/schemas/index.md

```diff
@@ -1632,3 +2130,335 @@ Specific property types support a subset of [JSON Schema validations](https://to
+            "type": "PROFILE_MASTER"
+          },
+          "maxLength": 20,
+          "minLength": 1,
+          "mutability": "READ_WRITE",
+          "permissions": [
+            {
+              "action": "READ_WRITE",
+              "principal": "SELF"
+            }
+          ],
+          "required": true,
+          "scope": "NONE",
+          "title": "exampleCustomProperty",
+          "type": "string",
+          "unique": "UNIQUE_VALIDATED"
+        }
+      },
+      "required": [
+        "exampleCustomProperty"
+      ],
+      "type": "object"
+    }
+  }
+}
+```
+
+#### Group Profile base subschema
+
+All Okta-defined Profile properties are defined in a Profile subschema with the resolution scope `#base`. These properties can't be removed or edited.  Attempting to do so anyway will have no effect.
+
+The base Group Profile properties are as follows:
+
+| Property          | Description                                                                                                                  | DataType | Nullable | Unique | Readonly | MinLength | MaxLength |
+| ----------------- | ---------------------------------------------------------------------------------------------------------------------------- | -------- | -------- | ------ | -------- | --------- | --------- |
+| name              | unique identifier for the Group                                                                                              | String   | FALSE    | TRUE   | FALSE    |           | 1024      |
+| description       | human readable description of the Group                                                                                      | String   | TRUE     | FALSE  | FALSE    |           | 1024      |
+
+#### Group Profile custom subschema
+
+All custom Profile properties are defined in a Profile subschema with the resolution scope `#custom`.
+
+```json
+{
+  "definitions": {
+    "custom": {
+        "id": "#custom",
+        "type": "object",
+        "properties": {
+            "customPropertyName": {
+                "title": "Title of custom property",
+                "description": "Description of custom property",
+                "type": "string"
+            }
+        },
+        "required": []
+    }
+  }
+}
+```
+
+#### Group Profile Schema Property object
+
+Group Profile schema properties have the following standard [JSON Schema Draft 6](https://tools.ietf.org/html/draft-wright-json-schema-validation-01) properties:
+
+| Property                       | Description                                | DataType                                          | Nullable | Unique | Readonly |
+|:-------------------------------|:-------------------------------------------|:--------------------------------------------------|:---------|:-------|:---------|
+| title                          | user-defined display name for the property | String                                            | FALSE    | FALSE  | FALSE    |
```

Comment:
General comment for the Description column: Initial uppercase for the descriptions.